scripts/generate-react.js

```diff
@@ -17,12 +17,15 @@ export default React.memo ? React.memo(${component.name}) : ${component.name};
 
 declare const ${component.name}: MdiReactIconComponentType;
 export default ${component.name};
-`, () => `import { ComponentType } from 'react';
+`, () => `import { ComponentType, SVGProps } from 'react';
 
-export interface MdiReactIconProps {
+type AllSVGProps = SVGProps<SVGSVGElement>
+
+type ReservedProps = 'size' | 'width' | 'height' | 'fill' | 'viewBox'
```

Comment:
```suggestion
type ReservedProps = 'color' | 'size' | 'width' | 'height' | 'fill' | 'viewBox'
```
To avoid type merging with our own definition. `color` is already defined as `string` on `SVGProps`.

Reply:
Integrated.